.github/workflows/main.yaml

```diff
@@ -116,8 +113,11 @@ jobs:
           file: tests/reports/.coverage.lcov
 
   test-app:
-    if: false # disable until GPU supported nodes become available
-    runs-on: ${{ matrix.os }}
+    runs-on: linux-amd64-gpu-l4-latest-1
+    container:
+      image: ghcr.io/nvidia-holoscan/holoscan-cli-build:cuda126-u2204
+      env:
+        NVIDIA_VISIBLE_DEVICES: ${{ env.NVIDIA_VISIBLE_DEVICES }}
```

Comment:
By the way, these values are not needed on our runners anymore. You can safely remove them. I will be removing the section in our docs that mentions them soon.

Reply:
Okay! Thank you @ajschmidt8 .